src/content/app/species-selector/components/species-lozenge-display-selector/SpeciesLozengeDisplaySelector.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { FormEvent } from 'react';
+import { useAppDispatch, useAppSelector } from 'src/store';
+
+import SimpleSelect from 'src/shared/components/simple-select/SimpleSelect';
+
+import styles from './SpeciesLozengeDisplaySelector.module.css';
+import { setLozengeDisplaySelection } from 'src/content/app/species-selector/state/species-selector-general-slice/speciesSelectorGeneralSlice';
+import { getLozengeDisplaySelection } from '../../state/species-selector-general-slice/speciesSelectorGeneralSelectors';
+
+export enum LozengeOptionValues {
+  COMMON_ASSEMBLY = 'common_assembly',
+  COMMON_TYPE_ASSEMBLY = 'common_type_assembly',
+  SCIENTIFIC_ASSEMBLY = 'scientific_assembly',
+  SCIENTIFIC_TYPE_ASSEMBLY = 'scientific_type_assembly',
+  ACCESSION = 'accession'
+}
```

Comment:
Several boring remarks, in the order of importance

1) It looks like the five different species display options will become a site-wide concern. They will be used in species lozenges; they will be used in the dropdown; they will be kept in redux in order to work cross-site; and they will be stored in the long-term browser storage in order to be recovered when the side loads. This suggests that they should not be defined in a component. The most appropriate place for them is a redux slice, probably `speciesSelectorGeneralSlice`.
2) Names. Right now, `"common_assembly"` reads as "common assembly", and `"scientific_assembly"` reads as "scientific assembly", which is a bit confusing. What if we did:

```
'common-name_assembly-name',
'common-name_type_assembly-name',
'scientific-name_assembly-name',
'scientific-name_type_assembly-name',
'assembly-accession-id'
```

3) Also naming. Since these display options a site-wide, can we please remove the word "lozenge" when we refer to them? A lozenge is a word that describes a piece of UI (something that is elongated and has rounded corners, similar to a pill), rather than the kind of data that we want to encode. Could we call it something like "species name display options"?

3) Enums. There have been a lot of negative comments about enums ([example link 1](https://www.youtube.com/watch?v=jjMbPt_H3RQ), [example link 2](https://www.youtube.com/watch?v=0fTdCSH_QEU)). Enums are an early typescript feature, which doesn't have a direct mapping to modern javascript, has a number of shortcomings, and is mostly redundant anyway. Couldn't we just express those display options as a union type?


```ts
type SpeciesNameDisplayOption =
  | 'common-name_assembly-name'
  | 'common-name_type_assembly-name'
  | 'scientific-name_assembly-name'
  | 'scientific-name_type_assembly-name'
  | 'assembly-accession-id'
```

There is very little, if any, benefit that an enum would offer over a union type.